Documentation/ceph-fs-mirror-crd.md

```diff
@@ -0,0 +1,46 @@
+---
+title: FilesystemMirror CRD
+weight: 3500
+indent: true
+---
+
+# Ceph FilesystemMirror CRD
+
+Rook allows creation and updating fs-mirror daemon through the custom resource definitions (CRDs).
+CephFS images can be asynchronously mirrored between two Ceph clusters.
```

Comment:
CephFS images or filesystem or even subvolumes?

Reply:
I should remove this line, only the following sentences are relevant.